tasks-by-users/mmusk/02-second-task-css/style.css

```diff
@@ -99,114 +98,296 @@ p {
-    margin: 70px 0;
+.categories {
+  margin: 70px 0;
 }
 
-.col-3{
-    flex-basis: 30%;
+.col-3 {
+  flex-basis: 30%;
 
-    min-width: 250px;
-    margin-bottom: 30px; 
+  min-width: 250px;
+  margin-bottom: 30px;
 }
 
-.col-3 img{
-    width: 100%;
+.col-3 img {
+  width: 100%;
 }
 
-.small-container{
-    max-width: 1080px;
-    margin: auto;
-    padding-left: 25px;
-    padding-right: 25px;
+.small-container {
+  max-width: 1080px;
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
 }
 
-.col-4{
-    flex-basis: 25%;
+.col-4 {
+  flex-basis: 25%;
 
-    padding: 10px;
-    margin-bottom: 50px;
-    min-width: 200px;
+  padding: 10px;
+  margin-bottom: 50px;
+  min-width: 200px;
 
-    transition: transform 0.5s;
+  transition: transform 0.5s;
 }
 
-.col-4 img{
-    width: 100%;
+.col-4 img {
+  width: 100%;
 }
 
-.title{
-    text-align: center;
-    line-height: 60px;
+.title {
+  text-align: center;
+  line-height: 60px;
+
+  margin: 0 auto 80px;
 
-    margin: 0 auto 80px;
+  position: relative;
+
+  color: #555;
+}
 
-    position: relative;
+.title:after {
+  content: "";
+  background: #ff523b;
 
-    color: #555;
+  bottom: 0;
+  left: 50%;
+  width: 80px;
+  height: 5px;
+  border-radius: 5px;
 
+  position: absolute;
+
+  transform: translateX(-50%);
 }
 
-.title:after{
-    content: "";
-    background: #ff523b;
+.col-4 h4 {
+  font-weight: normal;
+  color: #555;
+}
 
-    bottom: 0;
-    left: 50%;
-    width: 80px;
-    height: 5px;
-    border-radius: 5px;
+.col-4 p {
+  font-size: 14px;
+}
 
-    position: absolute;
-    
-    transform: translateX(-50%);
+.ratings .bi {
+  color: #ff523b;
 }
 
-.col-4 h4{
-    font-weight: normal;
-    color: #555;
+.col-4:hover {
+  transform: translateY(-5px);
 }
 
-.col-4 p{
-    font-size: 14px;
+.col-2 .offer-img {
+  padding: 50px;
 }
 
-.ratings .bi{
-    color: #ff523b;
+.small {
+  color: #555;
 }
 
-.col-4:hover{
-    transform: translateY(-5px)
+.testimonials {
+  padding-top: 100px;
 }
 
-.col-2 .offer-img{
-    padding: 50px;
+.testimonials .col-3 {
+  text-align: center;
+
+  padding: 40px 20px;
+  box-shadow: 0 0 20px 0px rgba(0, 0, 0, 0.1);
+
+  transition: transform 0.5s;
 }
 
-.small{
-    color: #555;
+.testimonials .col-3 .img {
+  width: 50px;
+  margin-top: 20px;
+  border-radius: 50%;
 }
 
-.testimonials{
-    padding-top: 100px;
+.testimonials .col-3:hover {
+  transform: translateY(-10px);
 }
 
-.testimonials .col-3{
-    text-align: center;
+/* brands */
+.brands {
+  margin: 100px auto;
+}
+
+.brands .col-5 {
+  width: 160px;
+}
+
+.brands .col-5 img {
+  width: 100%;
+
+  cursor: pointer;
+
+  filter: grayscale(100%);
+}
+
+.brands .col-5 img:hover {
+  filter: grayscale(0);
+}
+
+/* footer */
+
+.footer {
+  background: #000;
+  color: #8a8a8a;
+
+  padding: 60px 0 20px;
+}
+
+.footer p {
+  color: #8a8a8a;
+}
+
+.footer h3 {
+  color: #fff;
+
+  margin-bottom: 20px;
+}
+
+.footer-col-1,
+.footer-col-2,
+.footer-col-3,
+.footer-col-4 {
```

Comment:
can you use attribute selector here @mmusk

Reply:
fixed @coder-mezab